powderprojection/MantidProjectionCalculator.py

```diff
@@ -0,0 +1,18 @@
+from powderprojection.ProjectionCalculator import ProjectionCalculator
+from mantid.simpleapi import SofQWNormalisedPolygon,mtd
+from math import sqrt,cos
+
+
+class MantidProjectionCalculator(ProjectionCalculator):
+    def CalculateProjections(self,input_workspace,output_workspace,qbinning,axis1,axis2):
```

Comment:
Should not be using upper case for function names see [PEP8](https://www.python.org/dev/peps/pep-0008/#function-names)